scripts/script.yaml

```diff
@@ -188,15 +188,18 @@
               options:
               - show: מצויין!
                 variable: feel_good
-                step:
-                - say: "שמחים לשמוע. בכל זאת נרצה לברר אם יש לך משהו מהתסמינים הללו:"
-                - goto: current-report-symptoms
-
               - show: לא כל כך טוב
                 variable: feel_bad
-                steps:
-                  - say: אוייש… איך זה בא לידי ביטוי?
-                  - goto: current-report-symptoms
+              - switch:
```

Comment:
a. all options need values, not variables
b. if both branches goto the same place, you don't need to repeat the goto.
c. the 'say' can remain inside the options' steps (no need for switch here)
d. a switch can't be one of the options either way